examples/variational_autoencoders/vae.py

```diff
@@ -17,25 +17,28 @@
 
 
 @zs.reuse('model')
-def vae(observed, n, x_dim, z_dim, n_particles):
+def vae(observed, x_dim, z_dim, n_z_per_x):
     with zs.BayesianNet(observed=observed) as model:
-        z_mean = tf.zeros([n, z_dim])
-        z = zs.Normal('z', z_mean, std=1., group_ndims=1, n_samples=n_particles)
+        z_mean = tf.zeros([1, z_dim])
```

Comment:
If 1 is used here, all x in the batch are sharing the same `n_z_per_x` samples of `z`?

Reply:
Ah yes, that is true ... hmm, I can also a `n` than and have both fair enough.